Internship2022TeamPatri/Internship2022TeamPatri/Controllers/Auth/Register/RegisterCustomView.swift

```diff
@@ -0,0 +1,41 @@
+//
```

Comment:
`RegisterCustomView.swift`, `RegisterCustomView.xib` & `Register.storyboard `should be in Views folder